Tools/autotest/sim_vehicle.py

```diff
@@ -350,6 +350,24 @@ def do_build_waf(opts, frame_options):
     os.chdir(old_dir)
 
 
+def do_build_parameters(vehicledir):
+    # build succeeded
+    # now build parameters
+    progress("Building fresh parameter descriptions")
+    old_dir = os.getcwd()
+    param_metadata_dir = os.path.join(find_root_dir(),
+                                      "Tools/autotest/param_metadata")
+    os.chdir(param_metadata_dir)
```

Comment:
I don't quite like that it will generate the documentation in this directory, I think it should generate in current directory.

Reply:
Changed.